config/olm/operatorgroup.yaml

```
@@ -3,7 +3,7 @@ apiVersion: operators.coreos.com/v1
 kind: OperatorGroup
 metadata:
   name: meteor-operator
-  namespace: aicoe-meteor
+  namespace: meteor-system
```

We aren't using olm (yet) but I think this needs to be reverted, to match the current `aicoe-meteor` namespace:

```suggestion
  namespace: aicoe-meteor
```